packages/sanity/test/manifest/extractManifestRestore.test.ts

```diff
@@ -95,6 +96,20 @@ describe('Extract studio manifest', () => {
                 },
               ],
             }),
+            defineField({
```

Comment:
By adding this to this type, we know that serialising -> restoring to the origial schema works => we know globalRefs work correctly now.

Without the changes in this pr, this crashes.